File: AtCoder/abc206/A.kt
private fun readLn() = readLine()!!
private fun readDouble() = readLn().toDouble()

fun main() {
    val n = readDouble()
    var ans = "so-so"
    if ((n * 1.08).toInt() < 206) {
        ans = "Yay!"
    } else if ((n * 1.08).toInt() > 206) {
        ans = ":("
    }
    print(ans)
}

File: AtCoder/abc206/A.swift
private func readLn() -> String {
	return readLine() ?? " "
}

private func readDouble() -> Double {
	return Double(readLn()) ?? Double(0)
}

let n = readDouble()
var ans = "so-so"
let m = Int(n * 1.08)
if m < 206 {
	ans = "Yay!"
} else if m > 206 {
	ans = ":("
}
print(ans)
